style.css

```diff
@@ -44,13 +45,35 @@ li a{
     text-transform: uppercase;
 }
 
-h1{
+main{
+    position: relative;
+}
+
+img{
+    position:absolute;
+    right:0;
+    top:10vh;
+    height:650px;
+    width:900px;
+    object-fit: cover;
+    padding:10px;
+    border:1px solid #c9a356;
```

Comment:
The design showed a dashed border, you can change this to dashed to get the desired result.

Or if you want to get real fancy you can remove the border property and add in this property instead - it's a little trick to have more control over the dashes

background-image: url("data:image/svg+xml,%3csvg width='100%25' height='100%25' xmlns='http://www.w3.org/2000/svg'%3e%3crect width='100%25' height='100%25' fill='none' stroke='%23333' stroke-width='4' stroke-dasharray='6 14' stroke-dashoffset='0' stroke-linecap='square'/%3e%3c/svg%3e");

stroke-width is a good one to change and the dasharray is how many lines followed by how many gaps